CustomResourceDefinition/v1alpha2/bool/main.tf

```
@@ -0,0 +1,28 @@
+
+locals {
+  declared_default_field = contains(keys(var.manifest), "default")
+}
+
+output "schema" {
+  value = {
+    type    = "bool"
+    version = "v1"
+    subItem = null
+    validations = {
+      has_default_value = can(var.manifest.default) ? true : false
+      default_value     = try(var.manifest.default, null)
```

O schema deveria carregar o campo já em seu valor final. Com isso em mente, a validação se o campo default possui um valor válido, deveria estar aqui.
```suggestion
      default_value     = try(tobool(var.manifest.default), null)
```